common/src/jni/main/cpp/conscrypt/native_crypto.cpp

```diff
@@ -42,11 +27,38 @@
 #include <openssl/rsa.h>
 #include <openssl/ssl.h>
 #include <openssl/x509v3.h>
-#include <openssl/aead.h>
-
 #include <vector>
 
-using namespace conscrypt;
+#include "conscrypt/compat.h"
+#include "conscrypt/macros.h"
+#include <NetFd.h>
+#include <ScopedPrimitiveArray.h>
+#include <ScopedUtfChars.h>
+
+#include "conscrypt/app_data.h"
+#include "conscrypt/bio_input_stream.h"
+#include "conscrypt/bio_output_stream.h"
+#include "conscrypt/bio_stream.h"
+#include "conscrypt/compatibility_close_monitor.h"
+#include "conscrypt/errors.h"
+#include "conscrypt/jni_constants.h"
+#include "conscrypt/jni_util.h"
+#include "conscrypt/network_util.h"
+#include "conscrypt/openssl_error.h"
+#include "conscrypt/scoped_ssl_bio.h"
+
+using conscrypt::AppData;
+using conscrypt::BioInputStream;
+using conscrypt::BioOutputStream;
+using conscrypt::BioStream;
+using conscrypt::CompatibilityCloseMonitor;
+using conscrypt::Errors;
+using conscrypt::JniConstants;
+using conscrypt::JniUtil;
+using conscrypt::NativeCrypto;
+using conscrypt::NetworkUtil;
+using conscrypt::OpenSslError;
+using conscrypt::Trace;
```

Comment:
This is tricky ... we don't want the JNI methods to be in the conscrypt namespace.